### DISCUSSION TITLE
node upgrades

### DISCUSSION THREAD

**Discussion by PFC-developer** [strangelove-ventures/cosmos-operator#335] [General]
how does heighlinger/cosmos-operator handle chain upgrades?
is there a 'best practice' for this?

**Comment by DavidNix** [comment]
You must wait for the upgrade to halt the processes. Then change the image in `spec.podTemplate` to the new version and apply the changes.

We are considering a feature to allow for [scheduled upgrades](https://github.com/strangelove-ventures/cosmos-operator/issues/15). But, as of this writing, we cannot commit to a timeline.
